core/tests/protractor_utils/ProfilePage.js

```diff
@@ -82,13 +93,19 @@ var ProfilePage = function() {
     if (explorationCardsCount === 0) {
       throw new Error('There is no exploration card on this profile');
     }
-    expect(explorationCardsCount).toBeGreaterThan(0);
+    await waitFor.visibilityOf(
+      explorationCardsCount,
```

Comment:
This is a number, not an element. I think you want to waitFor the visibility of `allExplorationCardElements.first()`. You can also remove the `if (...) { throw new Error(...)}` code above, since that's also redundant with your waitFor

I think this is why the users e2e workflow was failing ([log](https://app.circleci.com/pipelines/github/oppia/oppia/19467/workflows/10a1aa44-d088-4636-ad71-93af47d0e6c1/jobs/109880)). Here's the error log you could have used to debug this:

```
Failures:
1) Visiting user profile page should show the explorations created by the user
  Message:
    Failed: Cannot read property 'bind' of undefined
  Stack:
    TypeError: Cannot read property 'bind' of undefined
        at ProtractorExpectedConditions.presenceOf (/home/circleci/oppia/node_modules/protractor/built/expectedConditions.js:341:40)
        at ProtractorExpectedConditions.visibilityOf (/home/circleci/oppia/node_modules/protractor/built/expectedConditions.js:381:30)
        at Object.visibilityOf (/home/circleci/oppia/core/tests/protractor_utils/waitFor.js:102:17)
        at ProfilePage.expectToHaveExplorationCards (/home/circleci/oppia/core/tests/protractor_utils/ProfilePage.js:96:19)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (internal/process/task_queues.js:93:5)
        at UserContext.<anonymous> (/home/circleci/oppia/core/tests/protractor_desktop/profileFeatures.js:165:5)
    From asynchronous test: 
    Error
        at Suite.<anonymous> (/home/circleci/oppia/core/tests/protractor_desktop/profileFeatures.js:160:3)
        at Object.<anonymous> (/home/circleci/oppia/core/tests/protractor_desktop/profileFeatures.js:124:1)
        at Module._compile (internal/modules/cjs/loader.js:1063:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
        at Module.load (internal/modules/cjs/loader.js:928:32)
        at Function.Module._load (internal/modules/cjs/loader.js:769:14)
```